libs/cosmos-sdk/baseapp/baseapp.go

```diff
@@ -903,6 +918,15 @@ func (app *BaseApp) runMsgs(ctx sdk.Context, msgs []sdk.Msg, mode runTxMode) (*s
 		//
 		// Note: Each message result's data must be length-prefixed in order to
 		// separate each result.
+
+		if isConvert {
+			txHash := tmtypes.Tx(ctx.TxBytes()).Hash(ctx.BlockHeight())
+			v, err := EvmResultConvert(txHash, msgResult.Data)
+			if err == nil {
+				msgResult.Data = v
+			}
+		}
+
 		events = events.AppendEvents(msgEvents)
```

Comment:
Do evm tx and cosmos tx event's attributes keep the same?

Reply:
May be developer can check by themselves

Reply:
yes